src/main/java/jenkins/plugins/awslogspublisher/AWSLogsHelper.java

```diff
@@ -5,16 +5,18 @@
 import com.amazonaws.retry.PredefinedRetryPolicies;
 import com.amazonaws.auth.AWSCredentials;
 import com.amazonaws.auth.AWSStaticCredentialsProvider;
-import com.amazonaws.auth.AWSCredentialsProvider;
-import com.amazonaws.auth.DefaultAWSCredentialsProviderChain;
```

Comment:
I believe the imports were removed because they were unused. It shouldn't cause any issues. I've tested with both explicitly provided keys as well as using an instance IAM role, and both worked for me.